Gemfile

```
@@ -27,7 +27,7 @@ path 'modules' do
   gem 'veteran_verification'
 end
 
-gem 'rails', '~> 6.1'
+gem 'rails', '>= 6.1.4.2'
```

We can just leave this how it currently is. running `bundle update rails` will update to the latest
```suggestion
gem 'rails', '~> 6.1'
```